src/entity/NotificationAttachment.ts

```diff
@@ -0,0 +1,29 @@
+import { Column, Entity, Index, ManyToOne, PrimaryColumn } from 'typeorm';
+import { Notification } from './Notification';
+
+export type NotificationAttachmentType = 'post';
```

Comment:
I'm actually thinking now, would we not want to opt to store the referenced post as well?
In the case of the post being deleted for instance?

Reply:
Good point!